app/components/trainees/record_details/view.html.erb

```diff
@@ -8,14 +8,14 @@
       action: govuk_link_to('Change<span class="govuk-visually-hidden"> trainee ID</span>'.html_safe, edit_trainee_trainee_id_path(trainee)),
     },
     trn_row,
+    trainee_status_row,
```

Comment:
As discussed, this should only be present if the trainee is deferred or withdrawn.